crates/nargo_cli/tests/test_data_ssa_refactor/brillig_array_eq/src/main.nr

```diff
@@ -0,0 +1,11 @@
+// Tests a very simple program.
+// 
+// The features being tested is using array equality on brillig
+fn main(x: [Field;3], y : [Field;3])  {
+    assert(array_eq(x, y));
+    assert(!array_eq(x, [1,2,4]));
```

Comment:
We probably want to add the edge case here, for empty arrays -- it should return true in that case.

An issue to open is to check if the ssa ir will give a compiler error on arrays of different lengths. If it gets to brillig ir, then it will panic in the `type_of_binary_operation` function

Reply:
Empty arrays don't seem to be allowed 
`error: Arrays must have at least one element`
And with different sizes the compiler issues
`error: Can only compare arrays of the same length. Here LHS is of length 3, and RHS is 2`